### PR TITLE
Maritime tutorials 💧 - Part 3 of 4

### PR DESCRIPTION
**Pull request by caguero**
## Summary

This pull request continues the series of maritime tutorials. I'm adding a new tutorial covering how to create a lander. I've also refactored the existing underwater tutorial a bit.

Screenshot from 2023-12-05 21-57-33

Pinging @crvogt for review.

## How to test it
```
colcon build --merge-install --packages-select gz-sim8 --cmake-target doc
```
Point your browser to `build/gz-sim8/doxygen/html/tutorials.html` and go from there.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: tutorials/files/underwater_vehicles/my_lrauv/model.sdf [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+              <normal_map>materials/textures/Tethys_Normal.png</normal_map>
+              <metalness_map>materials/textures/Tethys_Metalness.png</metalness_map>
+              <roughness_map>materials/textures/Tethys_Roughness.png</roughness_map>
+            </metal>
+          </pbr>
+          <script>
+            <uri>materials/scripts/</uri>
+            <uri>materials/textures/</uri>
+            <name>tethys/Tethys_Diffuse</name>
+          </script>
```
Comment (iche033) [inline tutorials/files/underwater_vehicles/my_lrauv/model.sdf:50]: this is used by gazebo-classic. I think we can remove these lines, as well as the material script `underwater_vehicles/my_lrauv/materials/scripts/model.material`

File: tutorials/files/underwater_vehicles/my_lrauv/model.sdf [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
```
Comment (iche033) [inline tutorials/files/underwater_vehicles/my_lrauv/model.sdf:102]: same as other comment, this is safe to remove

Comment (iche033) [reply tutorials/files/underwater_vehicles/my_lrauv/model.sdf:102]: and other `<script>` tags in this sdf file

File: tutorials/lander.md [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+
+```
+
+We can now test it in Gazebo. Launch it with:
+
+```bash
+gz sim ~/gazebo_maritime/models/my_lander/model.sdf
+```
+
+You should visualize your model as a box.
```
Comment (iche033) [inline tutorials/lander.md:88]:
```suggestion
You should see your model visualized as a box.
```

File: tutorials/lander.md [hunk 4]
```diff
@@ -0,0 +1,422 @@
...
+  <mQ>0</mQ>
+  <nRabsR>-632.698957</nRabsR>
+  <nR>0</nR>
+</plugin>
+```
+
+### Drop weight
+
+A common feature in maritime landers is to include a drop weight device. When
+this device is enabled, some part of the vehicle is dettached from the main
```
Comment (iche033) [inline tutorials/lander.md:338]:
```suggestion
this device is enabled, some part of the vehicle is detached from the main
```

File: tutorials/lander.md [hunk 5]
```diff
@@ -0,0 +1,422 @@
...
+chassis. At this point, the vehicle becomes positive buoyant, and starts moving
+up towards the water surface.
+
+In order to simulate this behavior, we'll need to make two changes to our lander
```
Comment (iche033) [inline tutorials/lander.md:342]: some places use `behaviour` while some `behavior`. Keep them consistent?

File: tutorials/files/underwater_vehicles/my_lrauv/model.sdf (hunk 2 above)
Comment (caguero) [reply tutorials/files/underwater_vehicles/my_lrauv/model.sdf]: Removed in [f667c7e](https://github.com/gazebosim/gz-sim/pull/2259/commits/f667c7e279672cee3b39b5b17fc4162ed20a5929).

File: tutorials/lander.md (hunk 5 above)
Comment (caguero) [reply tutorials/lander.md:342]: Updated in [f667c7e](https://github.com/gazebosim/gz-sim/pull/2259/commits/f667c7e279672cee3b39b5b17fc4162ed20a5929).

File: tutorials/lander.md [hunk 6]
```diff
@@ -0,0 +1,422 @@
...
+Create a workspace to store your brand new model named `my_lander`.
+
+```bash
+mkdir -p ~/gazebo_maritime/models/my_lander && cd ~/gazebo_maritime/models/my_lander
+```
+
+Download the `model.config` file and copy it within that directory:
+
+```bash
+wget https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/lander/model.config -o ~/gazebo_maritime/models/my_lander
```
Comment (crvogt) [inline tutorials/lander.md:40]:
```suggestion
wget https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/lander/model.config -o ~/gazebo_maritime/models/my_lander/model.config
```

File: tutorials/lander.md [hunk 7]
```diff
@@ -0,0 +1,422 @@
...
+@image html files/lander/simple_lander.png
+
+## Adding visuals
+
+Let's now add nicer-looking visuals to our lander. Download the following
+`COLLADA` mesh file from here.
+
+```bash
+mkdir ~/gazebo_maritime/models/my_lander/meshes
+wget https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/lander/inkfish-lander.dae -o ~/gazebo_maritime/models/my_lander/meshes
```
Comment (crvogt) [inline tutorials/lander.md:99]:
```suggestion
wget https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/lander/inkfish-lander.dae -o ~/gazebo_maritime/models/my_lander/meshes/inkfish-lander.dae
```

**Review by crvogt — APPROVED** [review]
Looks good.

File: tutorials/lander.md [hunk 8]
```diff
@@ -0,0 +1,422 @@
...
+          <box>
+            <size>1.325 1.1503 1.5</size>
+          </box>
+        </geometry>
+      </visual>
+    </link>
+
+  </model>
+</sdf>
+
```
Comment (ahcorde) [inline tutorials/lander.md:79]:
```suggestion
</sdf>
```

File: tutorials/lander.md [hunk 9]
```diff
@@ -0,0 +1,422 @@
...
+## Adding plugins
+
+### Buoyancy
+
+We are going to start extending our model adding buoyancy properties. For that,
+we need to do two things:
+
+
+1. Create a Gazebo world SDF file and load the buoyancy world plugin.
```
Comment (ahcorde) [inline tutorials/lander.md:151]:
```suggestion
we need to do two things:

1. Create a Gazebo world SDF file and load the buoyancy world plugin.
```

File: tutorials/underwater_vehicles.md [hunk 10]
```diff
@@ -1,201 +1,320 @@
...
+
+```bash
+mkdir -p ~/gazebo_maritime/models/my_lrauv && cd ~/gazebo_maritime/models/my_lrauv
+```
+
+Download all the files from here and copy them within that directory:
+
+```bash
+wget -r https://raw.githubusercontent.com/gazebosim/gz-sim/gz-sim8/tutorials/files/underwater_vehicle/my_lrauv -o ~/gazebo_maritime/models/my_lrauv
+
```
Comment (ahcorde) [inline tutorials/underwater_vehicles.md:41]:
```suggestion
wget -r https://raw.githubusercontent.com/gazebosim/gz-sim/gz-sim8/tutorials/files/underwater_vehicle/my_lrauv -o ~/gazebo_maritime/models/my_lrauv
```

**Review by ahcorde — APPROVED** [review]
Minor comments, LGTM

File: tutorials/lander.md (hunk 8 above)
Comment (caguero) [reply tutorials/lander.md]: Line removed in [2648ff4](https://github.com/gazebosim/gz-sim/pull/2259/commits/2648ff4f45e479dc7636bb6ff6cce1f4db6e2b0e).

**Pull request merged**